Doc/library/stdtypes.rst

```diff
@@ -609,6 +609,12 @@ class`. In addition, it provides a few more methods:
 
    .. versionadded:: 3.8
 
+.. method:: int.is_integer()
+
+   Returns ``True``.
```

Comment:
Perhaps it'd be a good idea to mention the `float` method below, since this description by itself could be fairly mystifying to users, since it'd be rather pointless by itself? Or alternatively, combine the definitions and move it plus `as_integer_ratio()` up to the common number operations section?